src/main/scala/com/fulcrumgenomics/umi/ExtractUmisFromBam.scala

```diff
@@ -232,8 +229,6 @@ object ExtractUmisFromBam {
     val molecularIndexBases = readStructureBases.filter(_.kind == SegmentType.MolecularBarcode).map(_.bases)
 
     // set the index tags
-    // TODO: when we remove the deprecated molecularBarcodeTags option, consider whether or not we still
-    //       need to have support for specifying a single tag via molecularIndexTags.
```

Comment:
Is the question in this comment sufficiently answered? I don't seen the vestigial evidence of `molecularBarcodeTags` so I'm not sure what behavior this is talking about.